libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -487,12 +509,36 @@ namespace SohImGui {
         else
             ImGui::Text(text, static_cast<int>(100 * val));
 
+        if(PlusMinusButton) {
+            std::string MinusBTNName = " - ##";
+            MinusBTNName += cvarName;
```

Comment:
Wouldn't it be better to use the names
plusMinusButton , minusBtnName, plusBtnName to stay on the naming convention ?